<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlterAddress</class>
 <widget class="QDialog" name="AlterAddress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#AlterAddress{
	background-image:url(img/alterAddress.jpg);
}</string>
  </property>
  <widget class="QPushButton" name="BtnConfirm">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newAddress">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>新  地  址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorInfo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;黑体&quot;;
color:rgb(255,255,0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
